restconnector/RequestHandler.cpp

```diff
@@ -83,7 +95,6 @@ void RequestHandler::processGET_( const ::zeq::Event& event )
 
     while( blocked_ )
     {
-        while( subscriber_->receive( 0 ) ){}
```

Comment:
I was assuming that this code was single threaded. If what you say is true, then the whole loop doesn't make a lot of sense to me, but I have to see the whole code to draw a conclusion.